### PR TITLE
Change: Use new release workflow and docker base images

### PR DESCRIPTION
**Pull request by timopollmeier**
## What
The workflows now use the new semantic version based workflows and the gvm-libs:oldstable base images.

## Why
This is part of updating the release process and the new gvm-libs images are needed for compatibility of the
debian packages.

## References

GEA-197

### CODE REVIEW

File: .docker/prod.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG VERSION=unstable
+ARG VERSION=oldstable
```
Comment (bjoernricks) [inline .docker/prod.Dockerfile:1]: I don't think this is correct here because we wont have a `greenbone/gsad-build:oldstable. Should be edge i guess.

Comment (timopollmeier) [reply .docker/prod.Dockerfile]: I've changed it  now so the gvm-libs base image and the build image have separate versions.

File: .github/workflows/release-pontos.yml [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
           github-user-token: ${{ secrets.GREENBONE_BOT_TOKEN }}
           gpg-key: ${{ secrets.GPG_KEY }}
           gpg-fingerprint: ${{ secrets.GPG_FINGERPRINT }}
           gpg-passphrase: ${{ secrets.GPG_PASSPHRASE }}
-          strategy: calendar
-          ref: ${{ env.RELEASE_REF }}
+          release-type: ${{ steps.release-type.outputs.release-type }}
+          release-version: ${{ inputs.release-version }}
+          ref: ${{ steps.release-type.outputs.release-ref }}
+          versioning-scheme: "semver"
```
Comment (bjoernricks) [inline .github/workflows/release-pontos.yml:55]: Looks god. Just a newline is missing at the end of the file.
```suggestion
          versioning-scheme: "semver"

```

File: .docker/prod.Dockerfile [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ARG VERSION=unstable
```
Comment (bjoernricks) [inline .docker/prod.Dockerfile:1]: Should be edge now right?
```suggestion
ARG VERSION=edge
```

Comment (timopollmeier) [reply .docker/prod.Dockerfile:1]: The build didn't work with edge but I can try changing it in another PR after this one is merged.

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**